webapp/src/ts/services/fast-action-button.service.ts

```diff
@@ -148,7 +155,7 @@ export class FastActionButtonService {
 
   getReportRightSideActions(context: ReportActionsContext): Promise<FastAction[]> {
     const actions = [
-      this.getSendMessageAction(context.communicationContext, { isPhoneRequired: true, useMailtoInMobile: true }),
+      this.getSendMessageAction(context.communicationContext!!, { isPhoneRequired: true, useMailtoInMobile: true }),
```

Comment:
I've never seen the `!!` after the expression (and Firefox does not like it). It's devilishly difficult to Google for (all examples I found are for _before_ the expression. Is this just casting it to a boolean? Why put it at the end rather than at the start? Are you trying to teach old dogs like me a new trick???

Reply:
Maybe we should use three just to be absolutely sure it's not null?

Thanks for the update! I thought I was losing my Googling skills...